Makefile

```diff
@@ -84,15 +91,15 @@ build-coverage: build_cmd=test -c -covermode=count -coverpkg ./pkg/controller/..
 build-coverage: clean $(CMDS)
 
 build-linux: build_cmd=build
-build-linux: arch_flags=GOOS=linux GOARCH=386
+build-linux: arch_flags=GOOS=linux GOARCH=$(GOARCH)
```

Comment:
These targets are only used for local development / test configuration. The build linux/amd64 binaries on the host, and are then copied into container images which run via kind or minikube.

If you want ppc64le binaries, you should get those just with `make build` with no changes the Makefile. Can you explain more what the goal of these changes is?

Reply:
when running the targets for run-local and run-console-local those do not get a build for the ppc64le arch. We would like to be able to do local development with ppc64le so upstreaming these local targets is the goal

Reply:
@ecordell is that ok to enable us to move forward?

Reply:
@ecordell wanted to see if this was still on your radar